examples/webcrypto/README.md

```diff
@@ -0,0 +1,11 @@
+# k6 webcrypto examples
+
+In this directory, you will find examples of how to use the k6's `webcrypto` module in your k6 scripts.
+
+> [!IMPORTANT]
+> We do run the tests based on these examples, that's why we have a simple convention for each example:
+>
+> * It should do any `console.log`. Since we try to detect that output (log) contain `INFO` keyword.
+> * It should NOT `try/catch` exceptions. Since we try to detect if keywords like `"Uncaught"` and `"ERRO"` should not appear in the output (logs).
```

Comment:
Based on [your comment](https://github.com/grafana/k6/pull/4188/files#r1919050525), I'd suggest the following for clarity, but I have no strong opinion. The current text is also fine, except for [the typo](https://github.com/grafana/k6/pull/4188/files#r1919001111).

```suggestion
> [!IMPORTANT]
> We do run the tests based on these examples; that's why we have a simple convention for each example:
>
> * Success condition: Example SHOULD output an `INFO` keyword using `console.log` at least once.
> * Failure condition: Example SHOULD NOT `try/catch` exceptions so that we can detect failures. The log output SHOULD NOT contain any keywords like `"Uncaught"` and `"ERRO"`.
```